### PR TITLE
FE document updates / SSR working for sample data

### PR DESCRIPTION
**Pull request by greypiglet**
- Continued document view
- Good test coverage
- Tidied some of the markup / modules so the names aren't misleading
- Moved tests into their own dirs

### CODE REVIEW

File: Comments/ClientApp/src/components/App/App.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// TODO: consider feeding these changes back to the design system
```
Comment (John-Williams) [inline Comments/ClientApp/src/components/App/App.scss]: i agree with this comment. these changes look like they belong in the design system.

File: Comments/ClientApp/src/components/CommentPanel/CommentPanel.scss [hunk 2]
```diff
@@ -1,31 +1,14 @@
...
-  right: 0;
-  top: 0;
-  width: 30vw;
-  z-index: 1;
-  height: 100vh;
   padding: rem($spacing-medium);
   &.inactive {
-    transform: translateX(30vw);
-    transition: transform 0.2s ease-out;
+    transform: translateX(240px) !important;
```
Comment (John-Williams) [inline Comments/ClientApp/src/components/CommentPanel/CommentPanel.scss]: pixels? why the change in unit?

File: Comments/ClientApp/src/components/App/App.scss (hunk 1 above)
Comment (greypiglet) [reply Comments/ClientApp/src/components/App/App.scss]: There are a couple of bits I've already found that need to be fed back up into the design system so I'll make a note.

File: Comments/ClientApp/src/components/CommentPanel/CommentPanel.scss (hunk 2 above)
Comment (greypiglet) [reply Comments/ClientApp/src/components/CommentPanel/CommentPanel.scss]: I thought maybe the comment panel would be of a fixed width rather than proportional - I'm sure this will change regardless as decisions are made!
